pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -2376,8 +2376,7 @@ func isOverlayNetworkInstance(netInstEntry *zconfig.NetworkInstanceConfig) bool
 	return netInstEntry.InstType == zconfig.ZNetworkInstType_ZnetInstMesh
 }
 
-func parseAppNetAdapterConfigEntry(
-	cfgApp *zconfig.AppInstanceConfig,
+func parseAppNetAdapterConfigEntry(cfgApp *zconfig.AppInstanceConfig,
```

Comment:
Looks like nothing changed here, not a big deal

Reply:
right, i used to add something, but removed that. need to realign the lines